detox/ios/DetoxXCUITestRunner/DetoxXCUITestRunner/Handlers/PredicateHandler.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PredicateHandler.swift (DetoxXCUITestRunner)
+//  Created by Asaf Korem (Wix.com) on 2024.
+//
+
+import Foundation
+import XCTest
+
+class PredicateHandler {
+  let springboardApp: XCUIApplication
```

Comment:
Is the `springboard` prefix on purpose? Can this be renamed to `app`? It's as though a general class (handler or whatever) is implemented over something very specific, but I don't fully understand what I'm seeing here.

Reply:
Yes, on purpose. Once it will support interactions with more apps, I will add them as params as well (e.g. `safariApp`, `appleLoginApp`)

Reply:
The name is to clarify that this is the springboard application.